src/pages/Home.tsx

```diff
@@ -1 +1,47 @@
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
 
+// Styled components
+const Container = styled.div`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	height: 100vh;
+	width: 100vw;
+	background-color: #f4f4f9;
+	padding: 20px;
+`;
+
+const Button = styled.button`
+	padding: 10px 20px;
+	margin: 10px;
+	font-size: 18px;
+	color: white;
+	background-color: #007bff;
+	border: none;
+	border-radius: 5px;
+	cursor: pointer;
+	width: 100%;
+	max-width: 200px;
+
+	&:hover {
+		background-color: #0056b3;
+	}
+`;
+
+function Home() {
+	const navigate = useNavigate();
+
+	return (
+		<Container>
+			{/* Knapp som navigerar till Start Game */}
+			<Button onClick={() => navigate("/start-game")}>Start Game</Button>
```

Comment:
Since this is a button where onClick doesn't do _anything else_ but navigate to another page, to be accessible, we should probably use a link, and style it as a button.

This is done something like this:

1) Import the Link component from react-router-dom
```
import { Link } from "react-router-dom";
```

2) Overwrite the Link component's styles with StyledComponents
```
const LinkButton = styled(Link)`
	padding: 10px 20px;
	margin: 10px;
	font-size: 18px;
	color: white;
	background-color: #007bff;
	border: none;
	border-radius: 5px;
	cursor: pointer;
	width: 100%;
	max-width: 200px;
	&:hover {
		background-color: #0056b3;
	}
`
```
You can read a little more about it [here](https://styled-components.com/docs/basics#styling-any-component) under "Styling any component" 

3) Use your new LinkButton and replace `onClick={() => navigate("/start-game")}` with `to="/start-game"`
```
<LinkButton to="/start-game">Start Game</LinkButton>
```

4) Clean up:
* Remove `useNavigate`
* Change the other Button (might need to set `to=""` since `to` is a required Prop on Link
* Will probably need to update the styling slightly since it will now be an `a`-tag instead of a `button` so there might be some default-styles (like `display: inline`) that we will need to override.

The upside of this:
The page will look exactly the same but our links will now be `a`-tags instead of `button`-tags so it will be more semantically correct.